src/pk_core.c

```diff
@@ -906,6 +906,66 @@ DEF(stdMathTangent,
   RET(VAR_NUM(tan(rad)));
 }
 
+DEF(stdMathSinh,
+  "sinh(rad:num) -> num\n"
+  "Return the hyperbolic sine value of the argument [rad] which is an angle "
+  "expressed in radians.") {
+
+  double rad;
+  if (!validateNumeric(vm, ARG(1), &rad, "Argument 1")) return;
+  RET(VAR_NUM(sinh(rad)));
+}
+
+DEF(stdMathCosh,
+  "cosh(rad:num) -> num\n"
+  "Return the hyperbolic cosine value of the argument [rad] which is "
+  "an angle expressed in radians.") {
+
+  double rad;
+  if (!validateNumeric(vm, ARG(1), &rad, "Argument 1")) return;
+  RET(VAR_NUM(cosh(rad)));
+}
+
+DEF(stdMathTanh,
+  "tanh(rad:num) -> num\n"
+  "Return the hyperbolic tangent value of the argument [rad] which is "
+  "an angle expressed in radians.") {
+
+  double rad;
+  if (!validateNumeric(vm, ARG(1), &rad, "Argument 1")) return;
+  RET(VAR_NUM(tanh(rad)));
+}
+
+DEF(stdMathArcSine,
+  "asin(rad:num) -> num\n"
+  "Return the arcsine value of the argument [rad] which is an angle "
+  "expressed in radians.") {
+
+  double rad;
+  if (!validateNumeric(vm, ARG(1), &rad, "Argument 1")) return;
```

Comment:
Check if the range is `[-1, +1]` and if it's not set a runtime error. (only for `asin` and `acos`).

Reply:
I wish we had something like;

```
i = 0
i in range(-1, 2)
```

But I'll go with a manual check for now.